src/Controller/AddOnOptionsController.py

```diff
@@ -398,29 +437,29 @@ def fillTables(self):
         #             self.table_roi.setItem(i, 2, items[2])
         #         i += 1
 
-        # patient hash ID table, which is just for displaying all the patients annonymized byt the software since intallation
-        with open('src/data/csv/patientHash.csv', "r") as fileInput:
+        # patient hash ID table, which is just for displaying all the patients anonymized byt the software since intallation
+        with open("src/data/csv/patientHash.csv", "r") as fileInput:
             next(fileInput)
             i = 0
             for row in fileInput:
                 items = [
-                    QTableWidgetItem(str(item.replace('\n', '')))
-                    for item in row.split(',')
+                    QTableWidgetItem(str(item.replace("\n", "")))
+                    for item in row.split(",")
                 ]
-                if len(items) > 2:
+                if len(items) >= 2:
```

Comment:
there are only two items currently in the reidentification spreadsheet.
nothing was being populated...